core/meta/expression-validation-rule.mjson

```diff
@@ -0,0 +1,50 @@
+{
+    "expression_validation_rule_criteria": {
+        "prototype": "core/meta/property-descriptor",
```

Comment:
This is the "old" ways of serializing property descriptors, back when they were called blueprints. Serializing "objectDescriptor" isn't necessary anymore, it creates noise and is unnecessary. Also, serializing properties with their default value isn't needed, like cardinality, the default value is 1. mandatory is still supported but it's been renamed isMandatory. Look at more recent object descriptors like Organization for examples

Reply:
also the name used for the property labels are particularly verbose. I've been using the same name as the property, can'y beat that conciseness and guaranteed to be unique in the file!